microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+i => finalDispatch
+ -->
+
+See also:
+
+* [Unminifying function names in DevTools performance profiles](https://blogs.windows.com/msedgedev/2022/02/03/unminifying-function-names-in-devtools-performance-profiles/) at the Microsoft Edge Blog.
+* [Unminify performance profiles (temp. PR link)](https://review.docs.microsoft.com/microsoft-edge/devtools-guide-chromium/evaluate-performance/unminify?branch=pr-en-us-1784).
+<!-- todo: update from review.dmc to live url -->
+
+
+<!-- ====================================================================== -->
+## Improved accessibility for Network Console and 3D View
+
+<!-- Title: Improvements for using assistive technology with DevTools -->
+<!-- Subtitle: Screen readers now announce better information in the Network Console and 3D View tools. -->
+
+In previous versions of Microsoft Edge, assistive technology announced incorrect information when providing a custom bearer token in the Network Console tool.  In Microsoft Edge version 99, this issue has been fixed.  When selecting the text box for providing a custom bearer token, assistive technology now announces "Token edit text Request View group".
+
+![The Network Console tool.](../../media/2022/03/network-console-tool.png)
+<!--
+1. Open DevTools.
+2. Press Ctrl+Shift+P and enter "Network Console".
+3. Open the Network Console tool.
+4. Select "Create a request".
+5. Select "Auth".
+6. Select "Bearer Token".
+7. Draw a red highlight around the "Token" edit textbox.
+8. If you can't see the "Token" edit textbox, drag the slider above "Hit 'Send' to issue the request" down to the bottom.
+-->
+
+In previous versions of Microsoft Edge, assistive technology would only announce the radio button text in the 3D View tool, without announcing the group name that the button belongs to.  In Microsoft Edge version 99, this has been fixed.  For example, when focus is on the **Use screen texture** radio button, assistive technology will now announce: "Color type radio group, Use screen texture, radio button, unselected".
+
+![The 3D View tool.](../../media/2022/03/3d-view-tool.png)
+<!--
+1. Open DevTools.
+2. Press Ctrl+Shift+P and enter "3D View".
+3. Open the 3D View tool.
+4. Select "DOM".
+5. Draw a red highlight around the "Color type" section and all the radio buttons in that section.
+-->
+
+To learn more about how using assistive technology with DevTools, read [Navigate DevTools with assistive technology](../../../accessibility/navigation.md).
+
+
+<!-- ====================================================================== -->
+## Improved source folder tree in the Sources panel
+
+In the **Page** tab of the **Sources** tool, the tree of source folders is now improved, with less clutter in the naming and outline of folders.  Unneeded path prefixes such as "../" and "./" have been removed.  The tree structure has been simplified by merging equivalent duplicate folders.  This was done by normalizing the absolute source URLs in the source maps.
```

Comment:
I don't think we need this detail about normalizing absolute source URLs. It feels to me like an implementation detail.
```suggestion
In the **Page** tab of the **Sources** tool, the tree of source folders is now improved, with less clutter in the naming and outline of folders.  Unneeded path prefixes such as `../` and `./` have been removed.  The tree structure has been simplified by merging equivalent duplicate folders. 
```